app/constants_es6.js

```diff
@@ -0,0 +1 @@
+export const STACK_NAME_VALIDATION_REGEX = '^[-_a-z0-9]+$';
```

Comment:
we have another file: `app/react/consants.ts`, let's use one

Reply:
yes, fixed.